build_html.sh

```diff
@@ -26,6 +26,7 @@ do
         printf '/:version/%s/* /:version/%s/404.html  404\n' "$lang" "$lang" >> build/html/_redirects
         printf '/:version/*    /:version/%s/:splat    301 Language=%s\n' "$lang" "$lang" >> build/html/_redirects
     fi
+    printf '/* /latest/en/404.html  404\n' >> build/html/_redirects
```

Comment:
are you sure this redirect is needed?

Reply:
Yes, for unknown versions, check out https://manual.mixxx.org/2.7/
I hope it works, I cannot test.

Reply:
Makes sense. Can add a comment above?

Reply:
```suggestion
    printf '/* /latest/404.html  404\n' >> build/html/_redirects
```
should work just as well.